modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java

```diff
@@ -1810,11 +1812,12 @@ private void _assertObjectDefinitions2() throws Exception {
 			_objectDefinitionLocalService.fetchObjectDefinition(
 				_group.getCompanyId(), "C_TestObjectDefinition4");
 
+		Assert.assertTrue(objectDefinition4.isAccountEntryRestricted());
 		Assert.assertTrue(
 			objectDefinition4.getAccountEntryRestrictedObjectFieldId() > 0);
-		Assert.assertTrue(objectDefinition4.isAccountEntryRestricted());
 		Assert.assertTrue(objectDefinition4.isEnableCategorization());
 		Assert.assertTrue(objectDefinition4.isEnableComments());
+		Assert.assertEquals("default", objectDefinition4.getStorageType());
```

Comment:
Please use the constant `ObjectDefinitionConstants#STORAGE_TYPE_DEFAULT` or method `ObjectDefinition#isDefaultStorageType`

I prefer this pattern `Assert.assertTrue(objectDefinition4.isDefaultStorageType());`